### PR TITLE
add in jetty-env gs gwc config

### PR DESCRIPTION
**Pull request by jeanmi151**
This need to wait for env var configuration in helm chart and docker deployments
connected with  https://github.com/georchestra/geoserver_minimal_datadir/pull/18

### CODE REVIEW

File: geoserver/jetty-env.xml [hunk 1]
```diff
@@ -35,5 +35,23 @@
        </New>
      </Arg>
    </New>
+   <New id="geoserver-gwc" class="org.eclipse.jetty.plus.jndi.Resource">
+     <Arg>jdbc/gwc</Arg>
+     <Arg>
+       <New class="org.apache.commons.dbcp.BasicDataSource">
+         <Set name="driverClassName">org.postgresql.Driver</Set>
+         <Set name="username"><Env name="JNDI_GEORCHESTRA_JDBC_USERNAME"/></Set>
```
Comment (fvanderbiest) [inline geoserver/jetty-env.xml:43]: The `JNDI_GEORCHESTRA_JDBC_` prefix is not explicit enough IMO.
Here we are configuring a db connection pool for geowebcache to store its cache usage statistics.

Something like `GWC_JNDI_PGUSER`, `GWC_JNDI_PGPASSWORD`, `GWC_JNDI__PGHOST`, `GWC_JNDI_PGPORT`, `GWC_JNDI_PGDATABASE` etc ... would suit better IMO.

Comment (jeanmi151) [reply geoserver/jetty-env.xml:43]: okay

**Review by edevosc2c — APPROVED** [review]

**Review by fvanderbiest — APPROVED** [review]

**Comment by edevosc2c** [comment]
Backport to 23?

**Comment by fvanderbiest** [comment]
> Backport to 23? Seems to be useful for isère.

ok

**Pull request merged**